samples/contrib/versioned-pipeline-ci-samples/helloworld-ci-sample/curl.bash

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+# strip gs:// prefix
+bucket_name=$(echo $3 | sed 's/gs:\/\///')
+data='{"name":'\""ci-$1"\"', "code_source_url": "https://github.com/kubeflow/pipelines/tree/'"$1"'", "package_url": {"pipeline_url": "https://storage.googleapis.com/'"$bucket_name"'/'"$1"'/pipeline.zip"}, 
```

Comment:
Sorry maybe a silly question. Should the `code_source_url` point to the user's repo or KFP's repo?

Reply:
code_source_url should points to the user's github repo, as part of source code management. Currently it only acts as a reminder, thus leaving it empty would also work for an example.